style.css

```diff
@@ -558,6 +578,14 @@ form.send-message button {
   color: #fff;
 }
 
+form.send-message button:focus-visible {
```

Comment:
Add a more contrasting outline when focusing form button with the keyboard for better visibility